everestrs/everestrs/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-mod schema;
+use everestrs_build::schema;
```

Comment:
Looks like this is the reason for the dependency inversion. Wouldn't it be possible to keep the schema implementations in everestrs and use them from everestrs_build, or is there a technical reason that I missed that necessitates moving these into everestrs?